templates/sign.steps.windows.yml

```diff
@@ -70,17 +73,21 @@ steps:
       ]
 - script: |
     set -o xtrace
-    for i in ${ARTIFACTS_DIR}/windows/*; do
-      zip_file="$(file ${i}/*.zip | cut -d: -f1)"
+
+    readarray -d '' artifacts < <(find ${ARTIFACTS_DIR} -maxdepth 1 -type f -regextype posix-egrep -regex '.*\.zip$' -print0)
+
+    for art in "${artifacts[@]}"; do
+      dir="$(dirname "$art")/extracted/$(basename "$art")"
+
+      zip_file="$art"
       if [ ! -f "${zip_file}" ]; then
         continue
       fi
 
-      zip -j -r "${zip_file}" "${i}" -x *.zip
-      for j in $(ls ${i}/*.exe); do
-          rm ${j}
-      done
+      zip -j -r "${zip_file}" "${dir}"
     done
+
+    rm -rf "$ARTIFACTS_DIR/extracted"
```

Comment:
```suggestion
    rm -rf "${ARTIFACTS_DIR}/extracted"
```

Reply:
Is this for aesthetics? A slash can't be part of a variable name, so there's no ambiguity to resolve with the braces.